gluten-core/src/main/java/org/apache/gluten/jni/JniUtils.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.gluten.init;
+package org.apache.gluten.jni;
```

Comment:
If we move it to `gluten-core`, could we do some minor refactors on this class at the same time? My suggestion would be:

1. `org.apache.gluten.jni.JniUtils` -> `org.apache.gluten.utils.ConfigUtil`
1. `JniUtils.toNativeConf` -> `ConfigUtil.serialize`

Thanks.